### DISCUSSION TITLE
Autocomplete change "No results" message

### DISCUSSION THREAD

**Discussion by charlesoestergaard** [heroui-inc/heroui#2231] [Q&A] [answered]
It would be awesome to be able to change the label of "No results found" of the autocomplete component, as our website is not in english.

**Comment by 0-don** [comment]
yes I need to change the translations for that

**Comment by eracoding-de** [comment]
Is there any workaround or is there a slot we can use on Autocomplete component?

**Comment by hiimkosu1** [comment, marked as answer]
There's a prop for this. Use it within Autocomplete component:

      listboxProps={{
        emptyContent: 'Your own empty Content text.'
      }}

**Reply by charlesoestergaard** [reply]
This shuld be added to the documentation at https://nextui.org/docs/components/autocomplete.

Thanks for the answer.

**Reply by leonkelvan** [reply]
> There's a prop for this. Use it within Autocomplete component:
> 
> ```
>   listboxProps={{
>     emptyContent: 'Your own empty Content text.'
>   }}
> ```

thanks for sharing, it worked.
